varats-core/varats/provider/bug/bug.py

```diff
@@ -100,141 +154,206 @@ def _get_all_issue_events(project_name: str) -> tp.List[IssueEvent]:
         A list of IssueEvent objects or None.
     """
 
-    def load_issue_events(github: Github) -> PaginatedList[IssueEvent]:
-        repository: Repository = github.get_repo(project_name)
-        return repository.get_issues_events()
+    github_repo_name = get_github_repo_name_for_project(
+        get_project_cls_by_name(project_name)
+    )
 
-    repo_path = get_local_project_git_path(project_name)
-    cache_file_name = repo_path.name.replace("/", "_") + "_issues_events"
+    if github_repo_name:
 
-    issue_events = get_cached_github_object_list(
-        cache_file_name, load_issue_events
-    )
+        def load_issue_events(github: Github) -> 'PaginatedList[IssueEvent]':
+            if github_repo_name:
+                repository: Repository = github.get_repo(github_repo_name)
+                return repository.get_issues_events()
+            raise AssertionError
+
+        cache_file_name = github_repo_name.replace("/", "_") + "_issues_events"
+
+        issue_events = get_cached_github_object_list(
+            cache_file_name, load_issue_events
+        )
 
-    if issue_events:
-        return issue_events
-    return []
+        if issue_events:
+            return issue_events
+        return []
+    # No github repo; Should not occur at this point
+    raise AssertionError
```

Comment:
```suggestion
    raise AssertionError(f"No github repo found for project {project_name}")
```
It's better to put the reasoning into the message. So a user get's more information when the assertion is triggered.